apps/llm-timeline/app/globals.css

```
@@ -2,26 +2,27 @@
-    --secondary: #f5f5f5;
+    --secondary: #f4f4ef;
     --secondary-foreground: #1a1a1a;
-    --border: #e5e5e5;
-    --input: #e5e5e5;
-    --ring: #a3a3a3;
-    --year-watermark: #e5e5e5;
+    --border: #e8e6de;
+    --input: #e8e6de;
+    --ring: oklch(70.5% 0.213 47.604);
+    --year-watermark: #e8e6de;
+    --font-sans: Inter, ui-sans-serif, system-ui, sans-serif;
```

Stylelint (17.10.0)

[error] Expected "Inter" to be "inter" (value-keyword-case)

(value-keyword-case)